src/test/java/com/google/api/codegen/testdata/go_example_library.baseline

```diff
@@ -20,7 +20,7 @@ package library_test
 import (
     "cloud.google.com/go/library/apiv1"
     "golang.org/x/net/context"
-    googleapis_example_library_v1 "google.golang.org/genproto/googleapis/example/library/v1"
+    pb "google.golang.org/genproto/googleapis/example/library/v1"
```

Comment:
`librarypb` would be okay, be it's common practice to make proto imports short, and they should always have the `pb` suffix.